[BUG] Problem user non root

### The bug

```
$ id
uid=1000(pi) gid=1000(pi)
```
```
2026/06/23 16:37:55 Server started at http://0.0.0.0:8100
├─ REST API:  http://0.0.0.0:8100/api/
└─ Dashboard: http://0.0.0.0:8100/_/
[INFO] 2026/06/23 16:38:00 wake.go:18: Wake triggered for RabbitHoleTV
[INFO] 2026/06/23 16:38:10 shutdown.go:18: Shutdown triggered for RabbitHoleTV
[ERROR] 2026/06/23 16:38:13 handlers.go:150: No user exists for uid 1000
```
```docker run -d --name upsnap -u 1000:1000 --restart=unless-stopped --cap-add CAP_NET_RAW --network=host -e UPSNAP_HTTP_LISTEN=0.0.0.0:8100 -v /home/pi/.config/upsnap:/app/pb_data seriousm4x/upsnap:latest
```

### The OS that UpSnap is running on

Raspbian OS (trixie)

### Version of UpSnap

latest

### Your docker-compose.yml content

```YAML
docker run -d --name upsnap -u 1000:1000 --restart=unless-stopped --cap-add CAP_NET_RAW --network=host -e UPSNAP_HTTP_LISTEN=0.0.0.0:8100 -v /home/pi/.config/upsnap:/app/pb_data seriousm4x/upsnap:latest
```

### Reproduction steps

```bash
n/a
```

### Additional information

n/a

--- a/Dockerfile
+++ b/Dockerfile
@@ -19,6 +19,8 @@ ENV UPSNAP_HTTP_LISTEN=${UPSNAP_HTTP_LISTEN}
 RUN apk update &&\
     apk add --no-cache tzdata ca-certificates nmap samba samba-common-tools openssh sshpass curl &&\
     rm -rf /var/cache/apk/*
+RUN addgroup -g 1000 &&\
+    adduser -G upsnap -D -H -u 1000 upsnap -s /sbin/nologin
 WORKDIR /app
 COPY --from=downloader /app/upsnap upsnap
 LABEL org.opencontainers.image.source="https://github.com/seriousm4x/UpSnap" \

